recipes/biofilm/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "biofilm" %}
+{% set version = "0.0.92" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biofilm-{{ version }}.tar.gz
+  sha256: 9cd7e1ab5e211618cfb7476d51b7a82045fa0292a3ea9cd0af685ede6ce3fa69
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or osx or py<38]
```

Comment:
Looks like this package declare some empty script, see this line in `setup.py`:

```
 scripts=[ ],
````

If we can get rid of that we can make it noarch. A patch and a PR upstream would do it ;-p

Reply:
Not sure I follow. auto-sklearn is available on Windows and OSX-64, it is a noarch package BTW.